### PR TITLE
Remove jQuery as dependency of web core js

### LINKED ISSUE #22: Get rid of jquery in core.js
Instead of getting a jquery object in the `initialize` method you should get a pure DOM Document.

If you still need a jQuery object you can wrap the dom document with it:

```js
initialize(el) {
    this.$el = $(el);
}
```

Its also better to know which components need jquery and which does not as every component itself need to import jquery correctly.

That the core.js is 100% clean of jquery the $.extend need to be replace: https://github.com/massiveart/web-js/blob/783b208d2a5208c409f1a75df613eb9474bb4272/src/core.js#L194

### PR DESCRIPTION
**Pull request by alexander-schranz**
fixes #22 

Allows to use web-js without jQuery.

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+initialize($el, options) {
+
+}
+```
+
+**After**
+
+```js
+initialize(el) {
+    var $el = el;
```
Comment (thomasduenser) [inline UPGRADE.md]: If it's no jQuery object anymore, shouldn't this be `var el = el;` or `var $el = $(el)`?

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+// or
+
+initialize($el, options) {
+    var $el = el;
```
Comment (thomasduenser) [inline UPGRADE.md]: Same here.

**Comment by thomasduenser** [comment]
@alexander-schranz else ACK

**Comment by dominikmatt** [comment]
@alexander-schranz ACK

File: UPGRADE.md (hunk 1 above)
Comment (alexander-schranz) [reply UPGRADE.md]: its an UPGRADE.md so I think your whole logic before is based that $el exist and is a jQuery Element.

**Comment by thomasduenser** [comment]
@alexander-schranz ACK

**Pull request merged**